### DISCUSSION TITLE
How Can I Disable Image and Css On Firefox?

### DISCUSSION THREAD

**Discussion by omerolmez** [php-webdriver/php-webdriver#775] [Q&A] [answered]
I didn't any guidance for disable image and css on firefox. How can I make? Thanks...

**Comment by andrewnicols** [comment, marked as answer]
Hi @omerolmez,

You can set any Firefox Preferences required for the instance by specifying them in the DesiredCapabilities during instantiation of the Driver.

See https://github.com/php-webdriver/php-webdriver/blob/master/lib/Remote/DesiredCapabilities.php#L284-L300 for an example.

You will need to look at the list of preferences (about:config) to find which preferences you want, if this is possible at all.
